algorithms/ar-kmp/python3/kmp algorithm python.py

```diff
@@ -0,0 +1,41 @@
+def KMPSearch(pat, txt): 
+	M = len(pat) 
+	N = len(txt) 
+	lps = [0]*M 
+	j = 0  
+	computeLPSArray(pat, M, lps) 
+
+	i = 0 
+	while i < N: 
+		if pat[j] == txt[i]: 
+			i += 1
+			j += 1
+
+		if j == M: 
+			print ("Found pattern at index " + str(i-j)) 
+			j = lps[j-1] 
+		elif i < N and pat[j] != txt[i]: 
+			if j != 0: 
+				j = lps[j-1] 
+			else: 
+				i += 1
+
+def computeLPSArray(pat, M, lps): 
+	len = 0
+	lps[0]

```

Comment:
Codacy found an issue: [Statement seems to have no effect](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6361667)